### PR TITLE
Copy message text button

### PR DESCRIPTION
**Pull request by CheckMC**
Adds a simple "Copy message text" button in the dropdown menu for messages. 

Issue: #1815

### CODE REVIEW

File: website/public/locales/en/message.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "confirm_open_link_body": "Do you want to open this link?",
   "confirm_open_link_header": "Confirm open link",
   "copy_message_id": "Copy message ID",
+  "copy_message_text": "Copy message text",
```
Comment (notmd) [inline website/public/locales/en/message.json:5]: Maybe something like "Copy raw message" is better here?

**Review by notmd — APPROVED** [review]
Thanks, lgtm

File: website/public/locales/en/message.json (hunk 1 above)
Comment (CheckMC) [reply website/public/locales/en/message.json:5]: That also works. I used "Copy message text" to keep it more in line with "Copy message ID", but raw text may be more accurate. will update

Comment (CheckMC) [reply website/public/locales/en/message.json]: changed

**Comment by notmd** [comment]
Thank you!

**Pull request merged**